lte/protos/session_manager.proto

```diff
@@ -763,6 +812,9 @@ message M5GSMSessionContext {
   string sm_pdu_dn_req = 22;               //SM PDU DN request container. Max size up to 255. TLV and 9.11.4.16.
   M5GSMCause m5gsm_cause = 23;
   QosRules requested_qos_rules = 24; //TS-24501 - 9.11.4.13 minimum 7 and max 65538, variable length.
+  TeidSet gnode_endpoint = 25; 
+  //Procedure transaction identity  
+  bytes procedure_trans_identity = 26;   //One byte TS-24501 9.6
```

Comment:
how is this used?

Reply:
Here is the defination of Teid 
TeidSet {
  bytes teid = 1;
  bytes end_ipv4_addr = 2;
}
Both teid and ip addr both are coming as bytes from AMF.

Reply:
Plan to close, unless any issue in using ?

Reply:
Described, same we will plan to use for UPF as well as par of config.